canonical-websites/www.ubuntu.com
# Fix link to the partners contact-us on the ai overview

## Done

- Fix link to the partners contact-us on the ai overview
- Updated the [copy doc](https://docs.google.com/document/d/1mJq8PxH8FVRVF9i1Xzh0f6M73eyjqZE4NaKYHRcRAKM/edit#)
- Added a redirect for /partners/contact-us

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/ai](http://0.0.0.0:8001/ai)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- See that the button 'Get in touch' in the partner section goes to the partner website's contact us form


## Issue / Card

Fixes #3520
